src/main/java/org/commonjava/indy/service/archive/services/ArchiveFileService.java

```diff
@@ -0,0 +1,41 @@
+package org.commonjava.indy.service.archive.services;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.io.*;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class ArchiveFileService {
```

Comment:
Little confused here. Seems this service is not used in this pr, so what's purpose of it?

Reply:
I'm planning to place zip file related function here